.github/dependabot.yml

```
@@ -0,0 +1,9 @@
+---
+version: 2
+
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+
+    schedule:
+      interval: daily
```

We should configure Dependabot to follow the commit message style:

```suggestion
      interval: daily
      
    commit-message:
      prefix: ci
```

(The prefix is `ci` since GitHub Actions updates are changes to CI configuration)